variables.tf

```diff
@@ -173,3 +173,9 @@ variable "task_role_permissions_boundary_arn" {
   default     = ""
   type        = string
 }
+
+variable "protocol_version" {
+  description = "The protocol (HTTP) version."
+  default     = "HTTP"
```

Comment:
According to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group the default should be "HTTP1" instead of "HTTP".